screen/.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2020": true,
+        "node": true
+    },
+    "extends": [
+        "standard"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 11
+    },
+    "rules": {
```

Comment:
Do you plan to use Prettier as well? If so, most of these rules should be in the Prettier config later instead.

Reply:
Nope, I think the linting part for components can be kept rather loose since it wil probably be a rather small codebase (compared to UI)